app_server/public/css/partials/_pagination.scss

```diff
@@ -0,0 +1,12 @@
+.pagination {
+  @extend .no-select;
+  li a {
+    cursor: pointer;
+  }
+  li.active a {
+    cursor: default;
+  }
+  li.disabled a {
+    cursor: default;
+  }
+}
```

Comment:
Not sure if this is generated (and also for some other files), would be nicer to follow the convention of adding new line at end of each file. There is probably some editor settings for this.